app/models/tag.rb

```diff
@@ -493,7 +493,7 @@ def self.autocomplete_fandom_lookup(options = {})
   def self.find_by_name(string)
     return unless string.is_a? String
     string = string.gsub('*s*', '/').gsub('*a*', '&').gsub('*d*', '.').gsub('*q*', '?').gsub('*h*', '#')
-    self.where('name = ?', string).first
+    self.where('tags.name = ?', string).first
```

Comment:
This one is deliberate, and I think Hound is wrong here, given what the method is trying to do.